.travis.yml

```diff
@@ -19,10 +19,8 @@ cache:
 
 addons:
   apt:
-    sources:
-      - ubuntu-toolchain-r-test
+    update: true
```

Comment:
Why are we setting this explicitly to true now?

Reply:
Honestly, Travis is weird. Compare these two tests: https://travis-ci.org/getsentry/sentry/builds/380386029 and https://travis-ci.org/getsentry/sentry/builds/380426007.

In the former, all jobs are updating the apt repos even without `update: true`, likely some automatic behavior because `addons/apt/packages` is defined, but strangely, some jobs fail while others pass with seemingly the same repo update in the logs. The latter has `update: true` added as suggested by Travis:

```
Running apt-get update by default has been disabled.
You can opt into running apt-get update by setting this in your .travis.yml file:
  addons:
    apt:
      update: true
```
... and it seems to make everything work.

Omakase (trust the chef).

Reply:
Weird that it works in some and not others though. Oh well.